Rakefile

```diff
@@ -22,3 +22,31 @@ Rake::TestTask.new(:test) do |t|
 end
 
 task default: :test
+
+desc "Deploy from a running instance. "
+task :deploy, %i(stack revision) => :environment do |_t, args|
```

Comment:
Will this Rake task be bundled with the Shipit gem? Would also be nice if it was namespaced i.e. `shipit::deploy`

Reply:
yes, but we will have to load it in the shipit rakefile to use it